build/doctum-checkout.conf.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+*
+* This file is part of the phpBB Forum Software package.
+*
+* @copyright (c) phpBB Limited <https://www.phpbb.com>
+* @license GNU General Public License, version 2 (GPL-2.0)
+*
+* For full copyright and license information, please see
+* the docs/CREDITS.txt file.
+*
+*/
+
+$iterator = Symfony\Component\Finder\Finder::create()
+	->files()
+	->name('*.php')
+	->in(__DIR__ . '/../phpBB/')
+	->notPath('#^cache/#')
+	->notPath('#^develop/#')
+	->notPath('#^ext/#')
+	->notPath('#^vendor/#')
+	->notPath('data');
+
+return new Doctum\Doctum($iterator, [
```

Comment:
Running with doctum-all.conf.php currently does not work as this config is not defined as the variable `$config` but passed directly to the constructor (see also previous version of this with sami). Once you've changed that, running with doctum-all.conf.php should also work fine.

Reply:
Sure, I missed that
Now fixed in 3be5ab3a5c87544c4dc7786cfb5c4dec609d0d0b, I added a comment to document this behavior